docs/docs/get-started/setup-lightdash/connect-project.mdx

```diff
@@ -85,19 +85,10 @@ you'll need to include the path to the sub-folder where your dbt project is (e.g
 
 :::caution Prerequisite
 
-If you're using a local dbt project, during the [install process](./install-lightdash.mdx) you should have set the path
-to your local dbt project.
+When you're using a local dbt project, during the [install process](./install-lightdash.mdx) provide the absolute path to your dbt project.
```

Comment:
When you're using a local dbt project, during the [install process](./install-lightdash.mdx), you should provide the absolute path to your dbt project.